### PR TITLE
feat: added api calls

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Live preview link: 
https://brand-factory-react.netlify.app/

### CODE REVIEW

File: src/api-Calls/api-utility.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { toast } from "react-toastify";
+
+const getData = async() =>{
+    try {
+        const res = await axios.get("/api/products")
+        const data = res.data.products;
+        console.log("data aaya hai", data);
+        return data;
+    } catch (error) {
+        console.log("product nhi aara : ", error);
```
Comment (pnchinmay) [inline src/api-Calls/api-utility.jsx:11]: Better remove these comments after making methods work

**Review by pnchinmay — APPROVED** [review]
Left a few sanity comments for good practices

File: src/api-Calls/api-utility.jsx (hunk 1 above)
Comment (bishnoimukesh) [reply src/api-Calls/api-utility.jsx:11]: Ok, I will.

**Review by xii-vi — APPROVED** [review]
Everything looks great
Just remove the comments pointed out by other approver.

**Pull request merged**